### PR TITLE
fix(ingest): raise specific error when generating token fails

### PR DESCRIPTION
**Pull request by petrjasek**
and ignore schedule when running ingest update via manage.py

SD-4434

### CODE REVIEW

File: superdesk/io/commands/update_ingest.py [hunk 1]
```diff
@@ -188,14 +188,14 @@ class UpdateIngest(superdesk.Command):
...
-            if not is_closed(provider) and is_service_and_parser_registered(provider) and is_scheduled(provider):
+            if not is_closed(provider) and is_service_and_parser_registered(provider):
                 kwargs = {
                     'provider': provider,
                     'rule_set': get_provider_rule_set(provider),
                     'routing_scheme': get_provider_routing_scheme(provider)
                 }
 
-                update_provider.apply_async(expires=get_task_ttl(provider), kwargs=kwargs)
+                update_provider(**kwargs)
```
Comment (ioanpocol) [inline superdesk/io/commands/update_ingest.py]: This will serialize all ingest providers updates for a single celery worker

Comment (petrjasek) [inline superdesk/io/commands/update_ingest.py]: it's not the celery task, but the command you can run via `python manage.py`

Comment (mdhaman) [inline superdesk/io/commands/update_ingest.py]: It will serialize all ingest providers.

Comment (petrjasek) [inline superdesk/io/commands/update_ingest.py]: right, reverted this

**Pull request merged**